### PR TITLE
Add dummy admin user

### PR DESCRIPTION
**Pull request by amitaibu**
I was looking for a way to login a user to a clean, local environment.
Currently there are no users, so PR currently creates an `admin` (I'll of crouse add a setting to control this option)

I was wondering what would be the best way to now login the user. Drush (Drupal's CLI) has a niffy feature where you can `drush user-login [user-name]` which will give you a one time login url (something like `http://example.com/user/reset/1/1472331807/BFc3e7OQhymw8zAENaw4PnzG0h59Ya4eZVyQxZedqLc`). I'd love a pointer on what you think would be the way forward.

Or, if you think it's not worth while, we can of course close it - I'm just doing it mostly for learning purposes 😄

### CODE REVIEW

**Comment by amitaibu** [comment]
Or maybe I could just enable the DummyLogin when some `debug` setting is enabled?

**Comment by snoyberg** [comment]
How about a setting to specify an identifier as an admin, and on first login check that setting and update the database?

File: Foundation.hs [hunk 1]
```diff
@@ -423,7 +424,8 @@ instance YesodAuth App where
             return ()
 
     authPlugins app =
-        [ authOpenId OPLocal []
+        [ authDummy
```
Comment (snoyberg) [inline Foundation.hs]: This can't be merged in as-is.

Comment (amitaibu) [inline Foundation.hs]: sure, this was just a WIP.

**Comment by amitaibu** [comment]
Closing in favor of #38, which is more focused on allowing user to login.

**Pull request closed without merging**